LICENSE

```diff
@@ -1,4 +1,4 @@
-node-graylog (C) 2011 Egor Egorov <me@egorfine.com> 
+Copyright (c) 2017 Wizcorp
```

Comment:
I have changed this, since there's literally nothing left of the original implementation.